backup_and_restore/index.adoc

```diff
@@ -27,3 +26,39 @@ You might run into several situations where {product-title}  does not work as ex
 * You have lost the majority of your control plane hosts, leading to etcd quorum loss.
 
 You can always recover from a disaster situation by xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restoring your cluster to its previous state] using the saved etcd snapshots.
+
+[id="application-backup-restore-operations-overview"]
+== Application backup and restore operations
+
+As a cluster administrator, you can back up and restore applications running on {product-title} by using the OpenShift API for Data Protection (OADP).
+
+OADP backs up and restores Kubernetes resources and internal images, at the granularity of a namespace, by using a supported version, 1.7, of link:https://velero.io/[Velero]. See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
```

Comment:
"at the granularity of a namespace"
I don't understand what this means.

"by using a supported version, 1.7, of link:https://velero.io/[Velero]"
This is a bit confusing. Seems to me that "supported version" and "1.7" don't work together. Can we just say "Valero"? 

"See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features]."
It's not clear that the preceding sentence was describing "OADP features", so the link text makes me pause. Perhaps something along the lines of "For more details of what OADP does, see link:blablabla [OADP features]."

Reply:
> "at the granularity of a namespace" I don't understand what this means.

Backup and restore can be used for cluster resources. Here, we want the user to know that they can back up a single namespace.

> "by using a supported version, 1.7, of link:[https://velero.io/[Velero]](https://velero.io/%5BVelero%5D)" This is a bit confusing. Seems to me that "supported version" and "1.7" don't work together. Can we just say "Valero"?

Not all versions of Velero are supported. Velero is an upstream open-source project that we have been using for MTC. That is why we have to specify that it is a supported version. I can rewrite this sentence to make it smoother.

> "See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features]." It's not clear that the preceding sentence was describing "OADP features", so the link text makes me pause. Perhaps something along the lines of "For more details of what OADP does, see link:blablabla [OADP features]."

The previous section was basic functionality. Features includes other functionality that the user might not use, for example, hooks and scheduling. Will check to see whether I can make it clearer.